### PR TITLE
Add wrapper in these classes to not affect in general components

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
Hi @vicnate5, I think that this case is not being covered by poshi tests. Please, could check this, because just noticed this after manage bar changes. (https://issues.liferay.com/browse/LPS-77284)

**Comment by vicnate5** [comment]
@alinedoleron It is correct that we do not have a poshi test for this yet. I can add one soon since this is a risk for regression.

I tested this manually and it looks good. Test results are OK too.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/form_portlet.js [hunk 1]
```diff
@@ -846,6 +846,8 @@ AUI.add(
 
 						instance._currentTab = 'formBuilder';
 
+						Liferay.fire('showForm');
```
Comment (rafaprax) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/form_portlet.js]: Hey @alinedoleron , please rename event to `showFormBuilder`. Thx

**Comment by alinedoleron** [comment]
sent to https://github.com/brianchandotcom/liferay-portal/pull/60684

**Pull request closed without merging**